### DISCUSSION TITLE
Stable release of 2.0.0

### DISCUSSION THREAD

**Discussion by dvzrv** [OpenAPITools/openapi-diff#281] [General]
Hi! I would like to package this software for Arch Linux.

When looking at the latest tags for this project, it appears to only consist of beta releases since 2018. Meanwhile the [2.0.0 milestone](https://github.com/OpenAPITools/openapi-diff/milestone/1) is finished. Is there a particular reason, why version 2.0.0 is not released?

I'm asking, because due to packaging policy I would not be able to package a beta of 2.0.0 and would have to go for version 1.2.0, which is from 2018.

**Comment by joschi** [comment]
@dvzrv Thanks for your interest in OpenAPI Diff!

I'd like to get the following issues and pull requests into OpenAPI Diff 2.0.0:

- Maven plugin (https://github.com/OpenAPITools/openapi-diff/issues/152 via https://github.com/OpenAPITools/openapi-diff/pull/278)
- Automate build and upload of Docker image (https://github.com/OpenAPITools/openapi-diff/pull/280)

After that I think we're good to go unless a blocker comes up.

**Reply by rhgiri** [reply]
Thanks. Is there a tentative date for 2.0.0 release please?

**Reply by rhgiri** [reply]
There isn't a non beta version available on Maven https://search.maven.org/artifact/org.openapitools.openapidiff/openapi-diff-core

Is there a way to get the 1.2.0 from 2018 that we can download?

**Reply by dvzrv** [reply]
Cool! As those two topics seem resolved now, I'm looking forward to a release! :)

**Reply by joschi** [reply]
> Is there a way to get the 1.2.0 from 2018 that we can download?

Yes, use the old Maven coordinates: https://github.com/OpenAPITools/openapi-diff/tree/1.2.0#maven

@rhgiri Is there any particular reason you want to use the old version instead of OpenAPI Diff 2.0.x?

**Reply by rhgiri** [reply]
I need a tagged version urgently and cant use a beta in production. The latest available tag is 1.2.0

When will 2.x tag be available please?

**Reply by joschi** [reply]
> I need a tagged version urgently and cant use a beta in production. The latest available tag is 1.2.0.

In the end it's just a string. I'd say OpenAPI-Diff 2.0.0-beta.X can be preferred over OpenAPI-Diff 1.1.0 or 1.2.0 simply because of the bugs which have been fixed in them.

**Comment by joschi** [comment]
OpenAPI-Diff 2.0.0 has been release: https://github.com/OpenAPITools/openapi-diff/releases/tag/2.0.0

Please see https://github.com/OpenAPITools/openapi-diff/discussions/297 for the related discussion.
